languages/ruby/exercises/concept/ostruct/.meta/design.md

```diff
@@ -0,0 +1,15 @@
+## Learning objectives
+
+- Understand how to require from stdlib
+- Understand OpenStruct
+- Learn &: block syntax
+
+## Concepts
+
+- `require`: Know how to use require
+- `ostruct`: Know how to use OpenStruct
+
+## Prerequisites
+
+- `enumeration`: Know how to consume hashes
```

Comment:
Just to be clear: was not listing the data types that are used (strings, integers) a conscious decision? As the exercise is a refactoring exercise that builds on the previous one, it is of course safe to assume that the previous exercise would depend on the type concepts.